tests/bootstrap.php

```diff
@@ -35,3 +52,29 @@
 if ($files) {
     require_once current($files);
 }
+
```

Comment:
trying this a little bit, this could be moved to `MediaControllerTest` like:
```php
public function testGetMediaAction(): void
{
    $kernel = new AppKernel();

    $application = new Application($kernel);
    $application->run(new StringInput(new ArrayInput([
        'command' => 'doctrine:database:drop',
        '--force',
    ])));
    $application->run(new ArrayInput([
        'command' => 'doctrine:database:create',
    ]));
    $application->run(new ArrayInput([
        'command' => 'doctrine:schema:create',
    ]));

    $client = new Client($kernel);
    $client->request('GET', '/api/media');
    $this->assertSame(200, $client->getResponse()->getStatusCode());
}
```

Reply:
bootstrap code only runs once for all tests right?

Reply:
yes, but IMO if you just want to run some unit test, this shouldn't be executed.